app.py

```diff
@@ -1,12 +1,17 @@
+from flask import Flask, url_for, request, redirect, flash, session, abort
```

Comment:
are these imports used for anything?

Reply:
Yeah I imported them for redirects, url_for is for dynamic linking. Abort is to raise errors just incase, request is to get things from models and stuff, etc.

Reply:
Flash is for flashing messages to the webpage like errors and info and session is to get things in the current session.

Reply:
if it's not actually being referenced anywhere in the code, it should be removed. Dead imports are confusing because when you see functions imported, you expect some functionality using them to exist, but there isn't. It only increases complexity without providing any value. This also applies to dependencies.

Reply:
Alright, I'll clean this up.